scripts/map_all_uptake_script.py

```diff
@@ -0,0 +1,58 @@
+"""Uptake of Scouting by local authority.
+
+This script produces a boundary report by local authority district, and plots
+the percentage of young people.
+
+This script has no command line options.
+"""
```

Comment:
Was this meant to be re-added? This is one of the files combined into map_uptake_script in the last branch

Reply:
No this wasn't meant to be readded, feel free to remove it